logexporter/cluster/log-dump.sh

```diff
@@ -831,14 +831,18 @@ function dump_node_info() {
     printf "%s\n" "${WINDOWS_NODE_NAMES[@]}" > "${nodes_dir}/windows_node_names.txt"
   fi
 
-  kubectl get nodes -o yaml > "${nodes_dir}/kubectl_get_nodes.yaml" || true
+  # If we are not able to reach the server, just bail out as the other
+  # kubectl calls below will fail anyway (we don't want to error out collecting logs)
+  kubectl version || return 0
+
+  kubectl get nodes -o yaml > "${nodes_dir}/kubectl_get_nodes.yaml"
```

Comment:
we already return from the method in line 836, so remove defensive check here and below.